card/spec/lib/card/set/trait_spec.rb

```diff
@@ -0,0 +1,62 @@
+
+describe Card::Set::Trait do
+  class Card
+    module Set
+      module Type
+        module Phrase
+          extend Card::Set
+          card_accessor :write, type: :pointer
+          card_accessor :read, type_id: Card::PointerID
```

Comment:
dumb question #4: we're not building a new layer on the permissions system are we?